operator/images/access-setup/content/bin/setup_work_dir.sh

```diff
@@ -203,8 +203,11 @@ tekton_results_manifest(){
       --dry-run=client -o yaml > "$results_db_secret"
 
     kubectl create secret generic -n tekton-results tekton-results-s3 \
-    --from-literal=S3_ACCESS_KEY_ID="$TEKTON_RESULTS_S3_USER" \
-    --from-literal=S3_SECRET_ACCESS_KEY="$TEKTON_RESULTS_S3_PASSWORD" \
+    --from-literal=aws_access_key_id="$TEKTON_RESULTS_S3_USER" \
+    --from-literal=aws_secret_access_key="$TEKTON_RESULTS_S3_PASSWORD" \
+    --from-literal=aws_region='us-east-1' \
```

Comment:
I feel like maybe the AWS region should not be hardcoded?

Reply:
this is only a "filling the gap" content for the dev env, we don't really use a s3 backend in this case but we use minio which simulate a s3 storage

Reply:
Then maybe put a value that describes that? E.g. `--from-literal=aws_region='not-applicable'`

Reply:
Done